apps/web/package.json

```diff
@@ -138,6 +138,7 @@
     "react-use-intercom": "1.5.1",
     "recoil": "^0.7.7",
     "remove-markdown": "^0.5.0",
+    "retell-sdk": "^4.40.0",
```

Comment:
Need this for calling retell ai API and validating request to webhook endpoint after the call to deduct credits